pages/spicedb/getting-started/coming-from/cancancan.mdx

```diff
@@ -0,0 +1,63 @@
+import PSDLight from '@/public/images/ps-light.svg'
+import PSDDark from '@/public/images/ps-dark.svg'
+import CCCLight from '@/public/images/ps-cancancan-light.svg'
+import CCCDark from '@/public/images/ps-cancancan-dark.svg'
+import SpiceDBLight from '@/public/images/ps-spicedb-light.svg'
+import SpiceDBDark from '@/public/images/ps-spicedb-dark.svg'
+
+
+# SpiceDB for Ruby on Rails CanCanCan users
+
+This document is designed to cover the conceptual differences between SpiceDB and the popular Ruby on Rails gem CanCanCan.
+
+The focus of the content below is not intended to be a competitive anlaysis, but rather a bridge to understand SpiceDB for existing Rails users.
```

Comment:
Might be helpful to visually call out this as well.

```suggestion
<Callout type="info" emoji="ℹ️">
The focus of the content below is not intended to be a competitive anlaysis, but rather a bridge to understand SpiceDB for existing Rails users.
</Callout>
```

and `import { Callout } from 'nextra/components'`